src/model/user.model.ts

```diff
@@ -76,10 +76,10 @@ export class User {
     @prop({ required: true, default: 0 })
     credit: number;
 
-    @prop({ default: "" })
+    @prop({ default: '' })
     evidenceInfo?: String;
 
-    @prop({ default: "" })
+    @prop({ default: '' })
     evidenceSentDate?: Date;
```

Comment:
I think these fields should not have default value since it is optional.

Reply:
ok